### ISSUE TITLE
ssl_cert: Allow forcing IPv4 or IPv6 check

### ISSUE THREAD

**Issue by z3ntu** [Icinga/icinga2#9979]
## Is your feature request related to a problem? Please describe.

check_ssl_cert supports passing `-4` and `-6` to force IPv4 or IPv6 respectively. There is no Icinga option to pass this through.

## Describe the solution you'd like

Add an option for Icinga to pass through these arguments.

## Describe alternatives you've considered

?

## Additional context

Something like #9978

**Comment by Al2Klimov** [comment]
Hello Luca!

Have you tried setting e.g. vars.check_ipv4?

Best,
A/K

**Comment by z3ntu** [comment]
No, is there any documentation what this does or how to use it? I can't seem to see any reference in the docs?

**Comment by Al2Klimov** [comment]
Indeed, docs are missing. But according to command-plugins.conf it's a boolean.

**Comment by Al2Klimov** [comment]
My suggestion seems not to be _the way_TM. A code grep for check_ipv[46] shows that lots of commands do this:

```
itl/command-plugins.conf:2142:	vars.check_ipv6 = "$dig_ipv6$"
```

They **provide own switches** on top of vars.check_ipv[46] in the first place and [document **them**](https://icinga.com/docs/icinga-2/latest/doc/10-icinga-template-library/#dig). So, strictly speaking: yes, the command lacks an own (documented) option.
